test/screenshot/golden.json

```diff
@@ -5,13 +5,13 @@
   "fab": "db36f52195c420062d91dd5ebe5432ad87247b3c1146fd547b0a195079bbce2f",
   "floating-label": "1d4d4f2e57e1769b14fc84985d1e6f53410c49aef41c9cf4fde94f938adefe57",
   "material-icon": "442b39fb22d2c7a74efb23ca098429b471501ce21df8662327bbf9871fe0bcb0",
-  "notched-outline": "3b132ebd457668970d16356de51d14f310497ba2ff6868ddb3e8637274b2fb35",
-  "select": "cef7604aff1573f4dfab18395598512a99fc9cd6c1cda7c2e2afd125809473d3",
+  "notched-outline": "7770dd381c27608a1f43b6f83da92507fe53963f5e4409bd73184b86275538fe",
+  "select": "ed9c021c8347f18e6ece48d55a8a3b30bb2380ec94276b002e9c064e7674e1c8",
   "tab": "0e53fa0ca9b2de4ff7941169a9b6a929a83b18e517c18404adeb40f7e644a2f1",
   "tab-indicator": "7ce7ce8fd50301c67d7ebfb0ba953208260ce2881bee0c7e640c46bf60dc90b6",
   "tab-scroller": "468866dd0c222b36b55485ab44a5760133a4ddfb2a6cf81e6ae4672d7e02a447",
-  "text-field": "d3582f0159959da922d0666ab6357b6641bf529b1c6613267f753625d5464ec3",
-  "text-field/helper-text": "dd523d5b6381a14be40f9c4994889ca908d317f7072b85b68be0c91b8b664b97",
+  "text-field": "2403e1d74007ec91da3ebb237257d5273652ebaca6a78c4a015d4658829adddc",
```

Comment:
since we removed the default version of Text Field in MDCW, we probably need to update the react code to reflect that.

Reply:
Ya - we had one extra set of text-fields (box). Also removed box class